modules/tinymce/src/core/main/ts/fmt/ExpandRange.ts

```diff
@@ -190,10 +186,12 @@ const findParentContainer = (
     if (!format[0].block_expand && dom.isBlock(parent)) {
       return parent;
     }
-
     // Walk left/right
     for (sibling = parent[siblingName]; sibling; sibling = sibling[siblingName]) {
-      if (!isBookmarkNode(sibling) && !isWhiteSpaceNode(sibling) && !isBogusBr(sibling)) {
+      const siblingIsNotSpace = NodeType.isText(sibling) && !onlyWhitespaces.test(sibling.data);
```

Comment:
This should probably use `Whitespace.isWhitespaceText(sibling)` instead. That'll also catch things like new line chars.

Reply:
Alternatively, you could use the `isWhiteSpaceNode(sibliling)` check that was here before, as the key part of this fix seems to be the `isEmpty` addition if I'm reading right.